docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
+
+### Before You Begin
+
+In this discussion, the [Chrome web browser](https://chrome.google.com/webstore/detail/cobalt/hekbjgfncacdinlajhgiakpaieajpfph/related) is used to illustrate the instructions to install and use the Cobalt wallet. 
+
+In a web browser, open the Cobalt web extension in the Chrome Web Store. Add the Cobalt extension. Then, click the Cobalt icon in the top-right corner of your browser.  The Cobalt Welcome! page appears.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-welcome.png" alt="Cobalt welcome" width="300" height="200" />
+</p>
+
+### Create an Account
+To begin, create an account for your wallet. Make sure that your password meets the following criteria:
+
+* Your password must contain at least 16-characters.
+* No common passwords are accepted, such as password, 12345, etc.
+* Your password cannot be saved to a keychain, therefore keep your password in a safe place.
+
+Perform the following steps:
+
+1. Enter a **password**. 
+2. Enter your password again to **confirm**.
+3. Click **Terms of Use**. After reading the terms and conditions, click the **checkbox** to agree on the terms.
+4. Click **Create**.
+
+
+### Create a New Wallet
+
+1. To create the new wallet, click **Create Wallet**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-getstarted.png" alt="Cobalt 2-sidechains" width="300" height="200" />
+</p>
+
+2. The **Choose a wallet** page appears. Click **Horizen EON** or **TokenMint** to create a wallet on a supported sidechain. 
+
+<p>
+<img src="/img/docs/get-started/cobalteon-choose-wallet.png" alt="Cobalt 2-sidechains" width="300" height="200" />
+</p>
+
+3. The **Create wallet for EON Testnet** page appears. Enter your wallet’s name. Click **Next**. 
+
+
+<p>
+<img src="/img/docs/get-started/cobalteon-wallet-name.png" alt="Cobalt WalletName" width="300" height="200" />
+</p>
+
+<p>
+<img src="/img/docs/get-started/cobalteon-wallet-name.png" alt="Cobalt Wallet Name" width="300" height="200" />
+</p>
+
+4. The **Seed Phrase** page appears. Write down your seed phrase. Click **Next**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-FriedSard-seed.png" alt="Cobalt Wallet Name" width="300" height="200" />
+</p>
+
+5. The **Confirm seed phrase** page appears. Select the word in the correct order to confirm your seed phrase. 
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-FSseed-confirm.png" alt="Cobalt SeedConfirm" width="300" height="200" />
+</p>
+
+6. Click **Confirm**. Cobalt displays your wallet name.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-wallet-complete.png" alt="Cobalt Wallet complete" width="300" height="200" />
+</p>
+
+### Import a Wallet
+
+To restore a previously created wallet, use the **Import Wallet** option. Before you restore a wallet, you need the wallet’s name, password, and seed phrase. 

```

Comment:
I would add at the beginning "Alternatively, you can import a wallet."

Reply:
Made the changed - "Alternatively, you can restore a previously created wallet by using the Import Wallet option."